lib/identity/design.rb

```diff
@@ -29,6 +29,14 @@ class Design < Sinatra::Base
         slim :"two-factor", layout: @layout
       end
 
+      get "/two-factor/recovery" do
+        slim :"account/two-factor/recovery", layout: @layout
```

Comment:
Sorry for the bikeshed, but mind explicitly setting the SMS number here?

``` ruby
@sms_number = fetch_sms_number
```

Or something like that?